onnx/onnxifi_loader.c

```diff
@@ -7,6 +7,11 @@
 #include <dlfcn.h>
 #endif
 
+/*
+ * temp enable ONNXIFI_ENABLE_EXT by all means
```

Comment:
i guess this is only for temporary test?

Reply:
Yeah, I will add this macro in CMAKE, so it is not ready to be merged now.